src/client/components/app/VaultDetail/VaultDetailPanels.tsx

```diff
@@ -317,18 +335,35 @@ export const VaultDetailPanels = ({
         </VaultActions>
       </Row>
 
-      {chartData && (
+      {chartData?.underlying && chartValue?.underlying && chartValue?.underlying !== '0' && (
         <VaultChart>
-          <StyledCardHeader header={t('vaultdetails:performance-panel.header')} />
+          <StyledCardHeaderContainer>
+            <StyledCardHeader header={t('vaultdetails:performance-panel.header')} />
+            <StyledChartTab active={selectedUnderlyingData === true} onClick={() => setSelectedUnderlyingData(true)}>
+              {selectedVault?.token?.symbol}
+            </StyledChartTab>
+            <StyledChartTab active={selectedUnderlyingData === false} onClick={() => setSelectedUnderlyingData(false)}>
+              USD
+            </StyledChartTab>
+          </StyledCardHeaderContainer>
 
           <ChartValueContainer>
             <ChartValueLabel>{t('vaultdetails:performance-panel.earnings-over-time')}</ChartValueLabel>
-            <ChartValue>{formatUsd(chartValue)}</ChartValue>
+            <ChartValue>
+              {selectedUnderlyingData ? (
```

Comment:
I think we should create a const in this cases to keep the html clean like in the example below I commented